ehcache-impl/src/test/java/org/ehcache/impl/internal/store/heap/OnHeapStoreBulkMethodsTest.java

```diff
@@ -86,8 +114,8 @@ public void testBulkComputeFunctionGetsValuesOfEntries() throws Exception {
       Map<Number, Number> newValues = new HashMap<>();
       for (Map.Entry<? extends Number, ? extends Number> entry : entries) {
         final Number currentValue = entry.getValue();
-        if(currentValue == null) {
-          if(entry.getKey().equals(4)) {
+        if (currentValue == null) {
```

Comment:
I know the existing formatting is wrong, but I'd rather not see formatting changes on code that is otherwise untouched.

Reply:
I have removed all formatting changes now